core/store/src/main/java/org/locationtech/geowave/core/store/metadata/AbstractGeoWavePersistence.java

```diff
@@ -97,6 +97,15 @@ public void removeAll() {
 		cache.invalidateAll();
 	}
 
+	public void remove(
```

Comment:
this method should be in IndexStoreImpl only as removeIndex, and it can simply delegate to `remove(ByteArrayId)`
see AdapterStoreImpl.removeAdapter as an example

Reply:
Made the recommended changes...the argument name for `remove(ByteArrayId)` is `adapterId` which threw me initially. I changed the name to `primaryId` to match the others